libplen/src/player.rs

```diff
@@ -287,13 +309,11 @@ impl Player {
                     self.id,
                     self.name.clone(),
                 );
-                (Some(ProjectileKind::from(new_bullet)), false)
-            } else {
-                (None, false)
+                return (Some(ProjectileKind::from(new_bullet)), false);
```

Comment:
Consider renaming `new_bullet` to `new_projectile`

Reply:
Fixed